docs/tools+libraries/tools/index.md

```diff
@@ -25,6 +25,7 @@ Miscellaneous tools
 -   sqlsharp: command line SQL client for most of the the System.Data based DB Connectors
 -   [Gendarme](/docs/tools+libraries/tools/gendarme/): Rule-based assembly analyser for developers.
 -   [Monoxide](/archived/monoxide): Interactive assembly analyser.
+-   [csharp](/repl): An interactive shell (REPL) for C#.
```

Comment:
Did you plan to include a page for the repl? If not, please remove the brackets for the markdown link.